src/wp-includes/blocks/navigation-area.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Server-side rendering of the `core/navigation-area` block.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Registers the `core/navigation-area` block on the server.
+ */
+function register_block_core_navigation_area() {
```

Comment:
`provides_context` is already defined in `block.json`. This PHP file isn't necessary. Everything could be handled through `block.json` file.

Reply:
We'll have to fix this in the Gutenberg plugin cc. @talldan @adamziel